### PR TITLE
SL for product analytics blog

### PR DESCRIPTION
**Pull request by Jstein77**
## What are you changing in this pull request and why?
Add dev blog on how to build a pipeline for a product launch using the semantic layer.

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/blog/2023-12-11-semantic-layer-on-semantic-layer.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+title: "How we built consitent product launch metrics with the dbt Semantic Layer."
+description: "We built and end-to-end data pipeline for measuring the launch of the dbt Semantic Layer using the dbt Semantic Layer."
+slug: product-analytics-pipeline-with-dbt-semantic-layer
+
+authors: [jordan_stein]
+
+tags: [dbt Cloud]
+hide_table_of_contents: false
+
+date: 2023-12-11
```
Comment (jasnonaz) [inline website/blog/2023-12-11-semantic-layer-on-semantic-layer.md]:
```SUGGESTION
date: 2023-12-12

File: website/blog/2023-12-11-semantic-layer-on-semantic-layer.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+Once the dust settles and the product is in the wild, the next thing the team needs to do is track how the product is doing. 
+How many users do we have? How is performance looking? What features are customers using? How often? Answering these questions is vital to understanding the success of any product launch.
+
+At dbt we recently made the [Semantic Layer Generally Available](https://www.getdbt.com/blog/new-dbt-cloud-features-announced-at-coalesce-2023). The Semantic Layer lets teams define business metrics centrally, in dbt, and access them in multiple analytics tools through our semantic layer APIs. 
+I’m a Product Manager on the Semantic Layer team, and the launch of the Semantic Layer put our team in an interesting, somewhat “meta,” position: we need to understand how a product launch is doing, and the product we just launched is designed to make defining and consuming metrics much more efficient.  It’s the perfect opportunity to put the semantic layer through its paces for product analytics. This blog post walks through the end-to-end process we used to set up product analytics for the dbt Semantic Layer using the dbt Semantic Layer. 
+
```
Comment (jasnonaz) [inline website/blog/2023-12-11-semantic-layer-on-semantic-layer.md]:
```SUGGESTION
<!-- truncate -->

File: website/blog/2023-12-11-semantic-layer-on-semantic-layer.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+![Screenshot 2023-09-28 at 4.05.25 PM.png](../static/img/blog/2023-12-11-semantic-layer-on-semantic-layer/Screenshot-dag.png)
+
+*Good old-fashioned debt DAG, with an extra node at the end*
+
+
+
+Let’s walk through the DAG from left to right: First, we have raw tables from the Semantic Layer Server loaded into our warehouse, next we have staging models where we apply business logic and finally a clean, normalized `fct_semantic_layer_queries` model. Finally, we built a semantic model named `semantic_layer_queries` on top of our normalized fact model. This is a typical DAG for a dbt project that contains semantic objects. Now let’s zoom in to the section of the DAG that contains our semantic layer objects and look in more detail at how we defined our semantic layer product metrics. 
+
+## How we build semantic models and metrics
```
Comment (jasnonaz) [inline website/blog/2023-12-11-semantic-layer-on-semantic-layer.md]: Do we want to link to the "How we build our metrics" best practices doc in this section? Bonus points if we remove the beta tag in that guide

File: website/blog/2023-12-11-semantic-layer-on-semantic-layer.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+metrics:
+  - name: queries
+    description: The total number of queries run
+    type: simple
+    label: Semantic Layer Queries
+    type_params:
+      measure: queries
+```
+
+*Queries metric configuration**
```
Comment (jasnonaz) [inline website/blog/2023-12-11-semantic-layer-on-semantic-layer.md]: Formatting is funky here

**Review by jasnonaz — CHANGES_REQUESTED** [review]
Delightful post - some light formatting adjustments then good to go

File: website/blog/2023-12-11-semantic-layer-on-semantic-layer.md (hunk 3 above)
Comment (Jstein77) [reply website/blog/2023-12-11-semantic-layer-on-semantic-layer.md]: Done! Where is the beta tag? I can't find it.

File: website/blog/2023-12-11-semantic-layer-on-semantic-layer.md (hunk 4 above)
Comment (Jstein77) [reply website/blog/2023-12-11-semantic-layer-on-semantic-layer.md]: Updated.

File: website/blog/2023-12-11-semantic-layer-on-semantic-layer.md (hunk 3 above)
Comment (mirnawong1) [reply website/blog/2023-12-11-semantic-layer-on-semantic-layer.md]: i'm lurking but Jason might mean [this `:::tip:::` callout](https://docs.getdbt.com/best-practices/how-we-build-our-metrics/semantic-layer-1-intro) @Jstein77

Comment (Jstein77) [reply website/blog/2023-12-11-semantic-layer-on-semantic-layer.md]: Ah I see it now. Removed!

**Review by jasnonaz — APPROVED** [review]
🚢

**Pull request merged**